frontend/src/FullPageRoutes/ContactUs.tsx

```diff
@@ -51,12 +52,17 @@ export default function ContactUs(): JSX.Element {
   const [values, setValues] = useState(contactValue)
   const classes = useStyles()
 
-  const setContactValue = (e: any) => {
+  const handleSubmit = (e: any) => {
     const { name, value } = e.target
     setValues({
       ...values,
       [name]: value,
     })
+    return request({
+      path: "/api/contact/",
+      method: "post",
+      body: contactValue,
+    }).then(e.preventDefault())
```

Comment:
Typically `e.preventDefault()` is used to ensure that the click event does not propagate. What are you trying to do here?